test/integration/requests/accountChannels.ts

```diff
@@ -0,0 +1,46 @@
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import { AccountChannelsRequest } from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('AccountChannels', function () {
```

Comment:
```suggestion
describe('account_channels', function () {
```

Reply:
To clarify something about naming conventions: we want to use exactly what rippled does if it's just a descriptive string or something, but if someone's actually going to be interacting with it in a JS way (such as a function param), then it should be camelCase.